queries/python/textobjects.scm

```diff
@@ -33,3 +32,7 @@
 
 (call) @call.outer
 (call (_) @call.inner)
+
+(parameters (_)? @parameter.inner) @parameter.outer
```

Comment:
Shouldn't parameter.outer be just the parameter with the comma?

Reply:
Will this also caputure line comments:
![image](https://user-images.githubusercontent.com/7189118/89134662-f985f200-d526-11ea-8d01-1fd71a68f5ee.png)

Reply:
I have made them more specific, but it got a little longer. Is there a way to capture a supertype?